<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewer</class>
 <widget class="QMainWindow" name="viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1390</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>264</width>
      <height>136</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="func_1_but">
       <property name="text">
        <string>sin^2(x) + cos^2(z) - y = 0</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="func_4_but">
       <property name="text">
        <string>sin(x) - cos(x) - y = 0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="func_2_but">
       <property name="text">
        <string>sin(x) * sin(z) - y = 0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="func_3_but">
       <property name="text">
        <string>sin(x)+sin(x) cos(z)+cos(z)-y =0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="func_5_but">
       <property name="text">
        <string>sqrt(sin(z)+cos(x)cos(z)+2)- y = 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>211</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>x angle:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>z angle:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="complex_rotation_but">
       <property name="text">
        <string>complex rotation</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="z_angle_text">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="x_rotation_but">
       <property name="text">
        <string>x rotation</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="x_angle_text">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>y angle:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="z_rotation_but">
       <property name="text">
        <string>z rotation</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="y_rotation_but">
       <property name="text">
        <string>y rotation</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="y_angle_text">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>211</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>1141</width>
      <height>681</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>211</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>z beg</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="x_end_text">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="x_beg_text">
       <property name="text">
        <string>-10</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>x beg</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>x end</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="z_beg_text">
       <property name="text">
        <string>-10</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>z end</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="z_end_text">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1390</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
